userguide/intro.rst

```diff
@@ -222,7 +222,8 @@ These screen options have changed:
   :menuselection:`Tasks --> Cloud Sync Tasks --> ADD`.
 
 * The :guilabel:`Name` field has been added to
-  :menuselection:`Tasks --> Replication Tasks --> Add`.
+  :menuselection:`Tasks --> Replication Tasks --> Add`. Replication
+  names must be unique.
```

Comment:
Be consistent in saying "Replication Tasks".  Technically, we don't have "Replication"s.
```suggestion
  Replication Task names must be unique.
```